client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -2156,9 +2156,8 @@
                                 scope.datelineDay = monthAndDay.day;
                                 scope.resetNumberOfDays(false);
                             }
+                            _.extend(item, updates);
```

Comment:
`var updates = {dateline: {}};` can be moved inside the inner if and then the 2 ifs can be combine in a single one.